web/templates/topic_sidebar_actions.hbs

```diff
@@ -41,7 +41,7 @@
     </li>
 
     <li>
-        <a tabindex="0" class="sidebar-popover-copy-link-to-topic" data-stream-id="{{ stream_id }}" data-topic-name="{{ topic_name }}">
+        <a tabindex="0" class="sidebar-popover-copy-link-to-topic" data-stream-id="{{ stream_id }}" data-topic-name="{{ topic_name }}" data-clipboard-text="{{ url }}">
```

Comment:
Getting the link of the topic for the clipboardJS inside the onMount instance was not working with the existing method. To make it easier to work with, a new attribute, data-clipboard-text, is added to the `Copy link to topic` anchor tag. This allows the clipboardJS to catch the URL. The value of data-clipboard-text is sent from `popover_menus.js` to the `topic_sidebar_actions` template.

However, I made this change in the same commit as migrating the `topic_menu`, since this is part of the migration. I also explained the reasoning behind it. Let me know if you want me to make a separate commit for this.

Reply:
I have noticed that we use similar patterns elsewhere in the codebase, so this appears to be a good solution.